doc/gov/chapters/chapter04.md

```diff
@@ -4,66 +4,106 @@
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 
-
 ## Table of Contents
 * [4.1 Introduction](#4.1)
 * [4.2 Workstreams Lead/Co-Lead Nomination and Selection](#4.2)
   * [4.2.1 Length of Terms](#4.2.1)
   * [4.2.2 Representation](#4.2.2)
+  * [4.2.3 Nomination](#4.2.4)
+  * [4.2.4 Selection](#4.2.4)
+  * [4.2.5 Interim Leads/Co-Leads](#4.2.5)
 * [4.3 Technical Steering Committee Lead/Co-Lead Nomination and Selection](#4.3)
-  * [4.3.1 Length of Terms](#4.3.1)
-  * [4.3.2 Representation](#4.3.2)
-  * [4.3.3 Membership and voting rights](#4.3.3)
 * [4.4 Governance Committee Lead/Co-Lead Nomination and Selection](#4.4)
-  * [4.4.1 Length of Terms](#4.4.1)
-  * [4.4.2 Representation](#4.4.2)
-  * [4.4.3 membership and voting rights](#4.4.3)
-* [4.5 Committee Structure](#4.5)
-* [4.6 Limits and Caps](#4.6)
-* [4.7 Tracking contributions](#4.7)
+* [4.5 Membership and voting rights](#4.5)
+* [4.6 Committee Structure](#4.6)
+* [4.7 Limits and Caps](#4.7)
+* [4.8 Tracking contributions](#4.8)
 
 <a name="4.1"></a>
 ## 4.1 Introduction
 
+The CNTT is structured into Workstreams so it can scale and accommodate participation from the wider community. The CNTT is open to participation by any individual or organization that agrees to comply with it's ToR (each such participant, a “Participant”). Each of the Workstreams within CNTT has one Lead and up to two Co-Leads. Leads and Co-Leads for each Workstream are selected by the community itself through nomination and voting processes. This document explains the CNTT policies and process in details regarding nomination and selection.
```

Comment:
ToR first use acronym needs expanded text.

Reply:
Also meybe a link to the ToR would be usefull.

Reply:
@pgoyal01 @CsatariGergely your feedback was addressed in here: https://github.com/cntt-n/CNTT/pull/896/commits/11f9f55716995b0d6fef82d5baf3c5e40733eb2c please confirm